CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+# Change Log
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/) and adheres to [Semantic Versioning](http://semver.org) as of version 1.0.0.
```

Comment:
I think that is generally a good idea, but it sort of only makes sense if you have defined a public API.

I'm not sure what we would consider the public API of jupiter-brain. Perhaps the CLI + env vars? However, it gets more nuanced. Does removing an env var require a major release?